rancilio-pid/rancilio-pid.ino

```diff
@@ -431,24 +433,23 @@ boolean checkSensor(float tempInput) {
   /********************************************************
     sensor error
   ******************************************************/
-  if ((tempInput < 0 || abs(tempInput - previousInput) > 25) && !sensorError) {
+  if ( ( tempInput < 0 || tempInput > 150 || abs(tempInput - previousInput) > 25) && !sensorError) {
     error++;
     sensorOK = false;
-    DEBUG_print("Error counter: ");
-    DEBUG_println(error);
-    DEBUG_print("temp delta: ");
-    DEBUG_println(tempInput);
+    sprintf(debugline, "WARN: temperature sensor reading: consec_errors=%d, temp_current=%f, temp_prev=%f", error, tempInput, previousInput);
+    DEBUG_println(debugline);
   } else if (tempInput > 0) {
     error = 0;
     sensorOK = true;
   }
   if (error >= maxErrorCounter && !sensorError) {
     sensorError = true ;
-    DEBUG_print("Sensor Error");
-    DEBUG_println(Input);
-  } else if (error == 0) {
```

Comment:
TempSensorRecovery config flag fehlt seltsamerweise in dem PR. Ich füge das hinzu.